spec/models/comment_spec.rb

```diff
@@ -72,7 +72,7 @@
         before { comment.save(validate: false) }
 
         it "changes the comment" do
-          expect(comment.update(comment_content: "Why did you block me?")).to be_truthy
+          expect(comment.update!(comment_content: "Why did you block me?")).to be_truthy
```

Comment:
Since we're using `update!` instead, we can drop the `expect(...).to be_truthy` (that's the other way to make it obvious when updates fail)